src/views/IssueChild.js

```diff
@@ -54,7 +54,7 @@ export default function IssueChild() {
   const { pointCursor } = usePointCursor();
 
   const _contracts = need.contracts(contracts);
-  const _point = need.point(pointCursor);
+  const _point = parseInt(need.point(pointCursor), 10);
```

Comment:
pointCursor should _always_ be `Maybe<number>` — where is it being set as a string?

Reply:
We weren't consistently using `lib/patp2dec` over `ob.patp2dec`. I've replaced all instances of the latter with the former, for consistency's sake.